Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM python:3.8-slim-buster AS intermediate
+ARG PYPI_INDEX_URL
+ENV URL=${PYPI_INDEX_URL}
+# comes as an argument for 'docker build --build-arg ... ' 
+RUN python -m pip install --upgrade pip
+RUN mkdir /root/.pip && touch /root/.pip/pip.conf && \
+    printf "[global]\nindex-url = ${URL}\n" > /root/.pip/pip.conf && \
+    cat /root/.pip/pip.conf
+RUN pip install myWallet==0.2
+
+FROM python:3.8-slim-buster
+RUN useradd --create-home appuser
+RUN chown -R appuser /home/appuser
+# path where pip looks for packages is '/usr/local/lib/python3.8/site-packages'
+COPY --from=intermediate /usr/local/lib/python3.8/site-packages /usr/local/lib/python3.8/site-packages 
+WORKDIR /home/appuser
+USER appuser
+COPY --chown=appuser:appuser requirements.txt requirements.txt
+RUN python -m pip install --upgrade pip
```

Comment:
It's better to chain Dockerfile RUN instructions with '&&' so we don't create too many layers- e.g:
```
RUN pip install myWallet==0.2 && \
 pip install --user -r requirements.txt && \
 rm /home/appuser/requirements.txt
```

Reply:
I know about layers, sorry, just forgot to chain it.